sklearn/preprocessing/imputation.py

```diff
@@ -123,6 +123,23 @@ def __init__(self, missing_values="NaN", strategy="mean",
         self.verbose = verbose
         self.copy = copy
 
+    def get_feature_names(self, input_features=None):
+        """
+        Return feature names for output features
+
+        Parameters
+        ----------
+        input_features : list of string, length len(input_features), optional
+            String names for input features if available. By default,
+            None is used.
+
+        Returns
+        -------
+        output_feature_names : list of string, length len(input_features)
+
+        """
+        return input_features
```

Comment:
What if some of the features had only `nan` in them? Shouldn't this method clip them from input_features, as `Imputer` does to the column itself?